platform/util/platform-util/src/test/java/org/codice/ddf/platform/util/XMLUtilsTest.java

```diff
@@ -100,6 +102,12 @@ public void testTransformNode() throws ParserConfigurationException, IOException
         assertThat(result.getWriter().toString(), is(equalTo(XML)));
     }
 
+    @Test
+    public void testGetRootNamespace() {
+        assertThat(XMLUtils.getRootNamespace(XML_WITH_NAMESPACE), is(equalTo("doggy-namespace")));
```

Comment:
It could be, and you definitely see most people do that. Personally, for short strings that are only used once, I prefer to just use the string itself since it is just as clear and fewer lines of code.